pkg/controller/issueapi/issue.go

```diff
@@ -127,6 +127,18 @@ func (ic *IssueAPI) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		controller.WriteJSON(w, http.StatusUnprocessableEntity, api.Error("invalid request: %v", err))
 		return
 	}
+	var realmID uint
+	if authApp != nil {
+		realmID = authApp.RealmID
+	} else {
+		// if it's a user logged in, we can pull realm from the context.
+		realm, err := controller.GetRealm(w, r, user)
+		if err != nil {
+			controller.WriteJSON(w, http.StatusUnprocessableEntity, api.Error("no realm selected"))
```

Comment:
How would a user specify their realm? I'm not sure it makes sense to extract the realm from the user? I actually don't understand this CUJ much.